immuni_exposure_ingestion/monitoring/celery.py

```diff
@@ -0,0 +1,38 @@
+from prometheus_client import Counter, Gauge
+
+from immuni_common.monitoring.core import NAMESPACE, Subsystem
+
```

Comment:
I'm tempted to keep consistency with the Mongo models, thus `BATCH_FILES_*` instead of `BATCHES_*`